core/src/main/java/com/netflix/conductor/service/MetadataServiceImpl.java

```diff
@@ -119,11 +122,22 @@ public void updateWorkflowDef(WorkflowDef workflowDef) {
     /**
      * @param workflowDefList Workflow definitions to be updated.
      */
-    public void updateWorkflowDef(List<WorkflowDef> workflowDefList) {
+    public BulkResponse updateWorkflowDef(List<WorkflowDef> workflowDefList) {
```

Comment:
Doesnt this change the behavior of the API? Previously, the API fails on the first error, but now it continues to process every item in the list. I suggest, we call this out in the release notes.

Reply:
yes, it does change the API behavior. I'll make sure we have that documented in the release note